test/plugin/utc.test.js

```diff
@@ -231,7 +231,7 @@ describe('UTC Offset', () => {
     const time = '2021-02-28 19:40:10'
     const hoursOffset = -8
     const daysJS = dayjs(time).utc().utcOffset(hoursOffset * 60, true)
-    const momentJS = moment(time).utc(true).utcOffset(hoursOffset, true)
```

Comment:
@iamkun I assume that this was not intentional and that you wanted to check if dayjs and moment return the same output given the same input.